ubenchmark/pom.xml

```diff
@@ -20,7 +20,7 @@ limitations under the License.
   <parent>
     <groupId>org.apache.calcite</groupId>
     <artifactId>calcite</artifactId>
-    <version>1.21.0-drill-r3</version>
+    <version>1.21.0-drill-r4</version>
```

Comment:
We might want to consider flatten maven plugin to keep the version in one place. But this is for other PR.
_upd:_ in future Calcite versions Gradle is used. So possibly this issue will gone